src/components/Card.jsx

```diff
@@ -1,10 +1,13 @@
+import './../styles/card.css';
+
 const Card = ({activeStep, handleNext, handleBack}) => {
     return (
-        <>
+        //will set up classname logic when I move to desktop styles
+        <div className="card-container">
```

Comment:
Typically, a card component is just a container component that accepts children as a prop. (i.e. The buttons should not be in your card component.) You could use composition to encapsulate the buttons.
```
// Page component will have something like this
<Card>
   <div>
     <Button label="Back" />
     <Button label="Next" />
   </div>
</Card>
```

Reply:
sorry, buttons aren't staying there - just needed them to test the step tracker. I'm already moving them into the footer component